meson_options.txt

```diff
@@ -44,3 +44,8 @@ option('xmp', type : 'feature',
 option('unitTests', type : 'feature',
   description : 'Build and run unit tests',
 )
+
+option('webready', type : 'boolean',
+  value: true,
```

Comment:
this is interesting. so if I understand correctly, webready is a bool that requires curl_dep to be found. I assume logic like this needs to be added.

Reply:
It's actually the other way around, curl depends on webready being enabled. Webready only (without curl) enables http access via the built-in implementation. If curl is enabled, http via curl takes precedence over the built-in http implementation.

Reply:
guess that logic should be added then.

```
if get_option('webready')
  curl_dep = dependency('libcurl', disabler: true, required: get_option('curl'))
else
  curl_dep = dependency('', disabled: true, required: false)
endif
```

or something like that.

Reply:
Makes sense, will add

Reply:
Added in the latest revision